libs/components/src/alert/stories/vl-alert.stories.ts

```diff
@@ -3,31 +3,43 @@ import '../vl-alert.component';
 import { ALERT_TYPE, ALERT_ICON } from '../vl-alert.model';
 import { alertArgs, alertArgTypes } from './vl-alert.stories-arg';
 import { Meta } from '@storybook/web-components';
-import { storyArgTypes, storyArgs } from '@domg-wc/common-storybook';
+import { story, storyArgTypes, storyArgs } from '@domg-wc/common-storybook';
+import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+import alertDoc from './vl-alert.stories-doc.mdx';
 
 export default {
     title: 'Components/alert',
     args: storyArgs(alertArgs),
     argTypes: storyArgTypes(alertArgTypes),
+    parameters: {
+        docs: {
+            page: alertDoc,
+        },
+    },
 } as Meta<typeof alertArgs>;
 
-const alertTemplate = ({ closable, icon, title, size, type, content }: typeof alertArgs) => html`
-    <vl-alert
-        ?data-vl-closable=${closable}
-        data-vl-icon=${icon}
-        data-vl-title=${title}
-        data-vl-size=${size}
-        data-vl-type=${type}
-        data-cy="alert"
-    >
-        <p>${content}</p>
-    </vl-alert>
-`;
+const AlertTemplate = story(
+    alertArgs,
+    ({ closable, icon, title, size, type, content, naked, message }) => html`
+        <vl-alert
+            ?data-vl-closable=${closable}
+            ?data-vl-naked=${naked}
+            data-vl-icon=${icon}
+            data-vl-title=${title}
+            data-vl-size=${size}
+            data-vl-type=${type}
+            data-vl-message=${message}
+            data-cy="alert"
+        >
+            ${unsafeHTML(content)}
+        </vl-alert>
+    `
+);
 
-export const alertDefault = alertTemplate.bind({}) as any;
```

Comment:
de stories die we exporteren wel in Pascal-case schrijven; dus `alertDefault` zou `AlertDefault` moeten worden en `alertError` zou `AlertError` moeten worden

we volgen hierbij een standaard van StoryBook